src/modules/NetworkController/public/New-SdnNetworkControllerCertificate.ps1

```diff
@@ -0,0 +1,277 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+function New-SdnNetworkControllerCertificate {
```

Comment:
Created this wrapper function based on new cert generation logic used in Start-SdnCertificateRotation and not rely on a working NC Cluster. Check this and see if it can be used as a unified version of new cert logic? @arudell